src/wallet/coin_selection.rs

```diff
@@ -69,6 +71,7 @@
 //!         Ok(CoinSelectionResult {
 //!             selected: all_utxos_selected,
 //!             fee_amount: fee_amount + additional_fees,
+//!             excess: Excess::Change { amount: 0, fee: 0 },
```

Comment:
This isn't the best in an example, because it probably creates an invalid tx (zero-value output for the change)

Reply:
Yes, I missed that. I can make `decide_change` public to compute the excess, but that would imply that it needs to be documented (I don't know if the function it's useful enough to be exposed in the docs) or I can mimic its behavior inside `AlwaysSpendEverything`.